bonita-integration-tests/bonita-integration-tests-client/src/main/java/org/bonitasoft/engine/business/data/HeapDumper.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (C) 2015 Bonitasoft S.A.
+ * Bonitasoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ **/
+
+package org.bonitasoft.engine.business.data;
+
+
+import java.io.File;
+import java.lang.management.ManagementFactory;
+import javax.management.MBeanServer;
+
+import com.sun.management.HotSpotDiagnosticMXBean;
+
+public class HeapDumper {
```

Comment:
I was wondering if we should keep it or not in order to use it when we want to make some heap dump in our tests.
What do you think?